src/armasters.py

```diff
@@ -34,3 +46,105 @@ def __init__(self, ndet):
         self._msbias_name = [None for all in xrange(ndet)]     # Master Bias Name
         self._mstrace_name = [None for all in xrange(ndet)]    # Master Trace Name
         self._mspixflat_name = [None for all in xrange(ndet)]  # Master Pixel Flat Name
+
+
+def master_name(mdir, ftype, setup):
+    """ Default filenames
+    Parameters
+    ----------
+    mdir : str
+      Master directory
+    ftype
+
+    Returns
+    -------
+
+    """
+    name_dict = dict(bias='{:s}/MasterBias_{:s}.fits'.format(mdir,setup),
+                     badpix='{:s}/MasterBadPix_{:s}.fits'.format(mdir,setup),
+                     trace='{:s}/MasterTrace_{:s}.fits'.format(mdir,setup),
+                     normpixflat='{:s}/MasterFlatField_{:s}.fits'.format(mdir,setup),
+                     arc='{:s}/MasterArc_{:s}.fits'.format(mdir,setup),
+                     wave='{:s}/MasterWave_{:s}.fits'.format(mdir,setup),
+                     tilts='{:s}/MasterTilts_{:s}.fits'.format(mdir,setup),
+                     )
+    return name_dict[ftype]
+
+def load_masters(slf, det, setup):
+    """ Save Master Frames
```

Comment:
I presume that you mean "Load Master Frames"  :-)  maybe include the parameters of the docstring as well (we'll have to go back and do these eventually anyway...